proto/src/lib.rs

```diff
@@ -96,7 +122,7 @@ mod tests {
         ($req:expr) => {{
             let _ = tracing_subscriber::fmt::try_init();
```

Comment:
Unwrap, since we're in tests?

Reply:
If you unwrap half the tests fail because tracing sub fails to setup. What happens is tracing sub "races" in all the tests and only one succeeds, and the rest error. 

So actually, we do need the let _ for all the ones that fail... it's a bit of a silly design on part of tracing.